docs/rules/newline-before-return.md

```diff
@@ -0,0 +1,77 @@
+# Require newline before `return` statement (newline-before-return)
+
+This rule was inspired by [Symfony2's Coding Standards][1]:
```

Comment:
This intro section should be about the concept behind the rule, not the rule itself. So, you should show some examples as to why a new line before return makes things easier to read. See: http://eslint.org/docs/rules/no-empty as an example